pyproject.toml

```diff
@@ -59,8 +59,7 @@ module = [
     "pytorch_lightning.loops.optimization.closure.py",
     "pytorch_lightning.loops.optimization.manual_loop.py",
     "pytorch_lightning.loops.evaluation_loop",
-    "pytorch_lightning.trainer.connectors.logger_connector.py",
-    "pytorch_lightning.trainer.connectors.logger_connector.fx_validator.py",
+    "pytorch_lightning.trainer.connectors.logger_connector.*",
```

Comment:
I think the list changed on master and now you will have to remove the logger_connector from the new list instead of adding it